MAUI/Cartesian-Charts/Trackball.md

```diff
@@ -220,6 +221,189 @@ this.Content = chart;
+        VerticalOptions = LayoutOptions.Center,
+        TextColor = Colors.White
+    };
+    label.SetBinding(Label.TextProperty, "DataItem.Values");
+
+    Image image = new Image
+    {
+        Source = "image.png",
+        WidthRequest = 40,
+        HeightRequest = 40
+    };
+
+    horizontalStackLayout.Add(label);
+    horizontalStackLayout.Add(image);
+
+    return horizontalStackLayout;
+});
+
+series1.TrackballLabelTemplate = trackballLabelTemplate;
+series1.TrackballLabelTemplate = trackballLabelTemplate;
+series1.TrackballLabelTemplate = trackballLabelTemplate;
+
+chart.Series.Add(series1);
+chart.Series.Add(series2);
+chart.Series.Add(series3);
+this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+![Trackball Label Template support in MAUI chart](Trackball_images/maui_chart_trackball_label_template.png)
+
+## Customize the Axis Label with DataTemplate
+
+Customize the appearance of axis label of trackball using [TrackballLabelTemplate[]() property of [ChartAxis](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html).
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    <chart:SfCartesianChart.Resources>
+            <DataTemplate x:Key="axisLabelTemplate">
+                <HorizontalStackLayout>
+                    <Label WidthRequest="50" HeightRequest="20" HorizontalTextAlignment="Center" BackgroundColor="Blue" Text="{Binding ValueX}" TextColor="White" FontSize ="15"/>
+                </HorizontalStackLayout>
+            </DataTemplate>
+    </chart:SfCartesianChart.Resources>
+
+   <chart:SfCartesianChart.TrackballBehavior>
+      <chart:ChartTrackballBehavior/>
+   </chart:SfCartesianChart.TrackballBehavior>
+
+    <chart:SfCartesianChart.XAxes>
+        <chart:DateTimeAxis TrackballLabelTemplate="{StaticResource axisLabelTemplate}" ShowTrackballLabel="True"/>
+    </chart:SfCartesianChart.XAxes>  
+
+    <chart:LineSeries ItemsSource="{Binding Data}" 
+                      XBindingPath="Name"
+                      YBindingPath="Values"/>
+                      
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+
+ChartTrackballBehavior trackball = new ChartTrackballBehavior();
+chart.TrackballBehavior= trackball;
+
+DateTimeAxis primaryAxis = new DateTimeAxis();
+primaryAxis.ShowTrackballLabel = true;
+
+DataTemplate axisLabelTemplate = new DataTemplate(() =>
+{
+    HorizontalStackLayout horizontalStackLayout = new HorizontalStackLayout();
+
+    var label = new Label
+    {
+        WidthRequest = 50,
+        HeightRequest = 20,
+        HorizontalTextAlignment = TextAlignment.Center,
+        BackgroundColor = Colors.Blue,
+        TextColor = Colors.White,
+        FontSize = 15
+    };
+    label.SetBinding(Label.TextProperty, "ValueX");
+
+    horizontalStackLayout.Add(label);
+
+    return horizontalStackLayout;
+});
+
+primaryAxis.TrackballLabelTemplate = trackballLabelTemplate;
```

Comment:
@EzhilarasanElangovan31 template name wrongly updated

Reply:
Updated the template name